frontend/packages/dev-console/src/components/pipelines/pipeline-builder/PipelineBuilderForm.tsx

```diff
@@ -86,67 +98,70 @@ const PipelineBuilderForm: React.FC<PipelineBuilderFormProps> = (props) => {
+    <YAMLEditorField name="yamlData" onSave={handleSubmit} schemaModel={PipelineModel} />
+  );
+
+  const sanitizeToForm = (newFormData: Pipeline) => {
+    const formData = {
+      name: newFormData.metadata?.name,
+      params: newFormData.spec?.params,
+      resources: newFormData.spec?.resources,
+      tasks: newFormData.spec?.tasks,
+      listTasks: values.formData.listTasks,
+    };
+    return _.merge({}, initialPipelineFormData, formData);
+  };
+
+  const sanitizeToYaml = () =>
+    safeJSToYAML(convertBuilderFormToPipeline(values.formData, namespace), 'yamlData', {
+      skipInvalid: true,
+    });
+
   return (
     <>
       <Stack className="odc-pipeline-builder-form">
         <StackItem>
-          <PipelineBuilderHeader existingPipeline={existingPipeline} namespace={namespace} />
-        </StackItem>
-        <StackItem isFilled className="odc-pipeline-builder-form__content">
-          <Form className="odc-pipeline-builder-form__grid" onSubmit={handleSubmit}>
-            <div className="odc-pipeline-builder-form__short-section">
-              <InputField
-                label="Name"
-                name="name"
-                type={TextInputTypes.text}
-                isDisabled={!!existingPipeline}
-                required
-              />
-            </div>
-
-            <div>
-              <h2>Tasks</h2>
-              <PipelineBuilderVisualization
-                namespace={namespace}
-                tasksInError={status?.tasks || {}}
-                onTaskSelection={(task, resource) => {
-                  setSelectedTask({
-                    taskIndex: values.tasks.findIndex(({ name }) => name === task.name),
-                    resource,
-                  });
-                }}
-                onUpdateTasks={(updatedTaskGroup, op) =>
-                  updateTasks(applyChange(updatedTaskGroup, op))
-                }
-                taskGroup={taskGroup}
-              />
-            </div>
-
-            <div>
-              <h2>Parameters</h2>
-              <PipelineParameters addLabel="Add Parameters" fieldName="params" />
-            </div>
-
-            <div>
-              <h2>Resources</h2>
-              <PipelineResources addLabel="Add Resources" fieldName="resources" />
-            </div>
-            <FormFooter
-              handleReset={closeSidebarAndHandleReset}
-              errorMessage={status?.submitError}
-              isSubmitting={isSubmitting}
-              submitLabel={existingPipeline ? 'Save' : 'Create'}
-              disableSubmit={
-                !dirty ||
-                !_.isEmpty(errors) ||
-                !_.isEmpty(status?.tasks) ||
-                values.tasks.length === 0
-              }
-              resetLabel="Cancel"
-              sticky
-            />
-          </Form>
+          <PipelineBuilderHeader />
         </StackItem>
+        <FlexForm className="odc-pipeline-builder-form__grid" onSubmit={handleSubmit}>
```

Comment:
Sidebar no longer is anchored:

![Screen Shot 2020-11-05 at 5 29 49 PM](https://user-images.githubusercontent.com/8126518/98304249-a3917200-1f8d-11eb-8da6-b5781551bf6d.png)

Reply:
Adding `overflow: auto` to the form fixed it.